provisioner/powershell/provisioner.go

```diff
@@ -501,6 +528,12 @@ func (p *Provisioner) generateElevatedRunner(command string) (uploadedPath strin
 		log.Printf("Elevated user %s converted to %s after escaping chars special to PowerShell",
 			p.config.ElevatedUser, escapedElevatedUser)
 	}
+	// Replace ElevatedPassword for winrm users who used this feature
+	p.config.ctx.Data = &EnvVarsTemplate{
```

Comment:
same here

Reply:
How would the UI for this work, setting the elevated password to an environment variable?